src/shared/api.ts

```diff
@@ -118,6 +174,9 @@ export const bedrockModels = {
 		supportsPromptCache: false,
 		inputPrice: 3.0,
 		outputPrice: 15.0,
+		cacheWritesPrice: 3.75, // per million tokens
+		cacheReadsPrice: 0.3, // per million tokens
```

Comment:
@Premshay out of curiosity where do you look up these values from?

Reply:
BTW, I've prepared infrastructure but haven't fully connected the Bedrock prompt caching feature. Reason is that it's still in preview for limited number of users. When this is fully released it'll be easy to apply. I also hope to integrate the new intelligent prompt routing when it is fully released and tested.